hack/common.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -e
+
+get_image_tag() {
+    local current_commit today
+    current_commit="$(git rev-parse HEAD)"
+    today="$(date +%Y%m%d)"
+    echo "v${today}-${current_commit:0:7}"
+}
+
+SSP_DIR="$(readlink -f $(dirname $0)/../)"
+WORK_DIR="/go/src/github.com/kubevirt/ssp-operator"
+REGISTRY=${REGISTRY:-quay.io/vparekh}
+REPOSITORY=${REPOSITORY:-ssp-builder}
+TAG=$(get_image_tag)
+BUILD_TAG="${REGISTRY}/${REPOSITORY}:${TAG}"
+BUILD_DIR=${SSP_DIR}/tests/build
```

Comment:
Maybe change this variable name to something like `TESTS_BUILD_DIR` to not confuse with code build dir